templates/master/00-master/baremetal/files/baremetal-haproxy.yaml

```diff
@@ -19,7 +19,10 @@ contents:
           path: "/etc/kubernetes/static-pod-resources/haproxy"
       - name: kubeconfig
         hostPath:
-          path: "/etc/kubernetes/kubeconfig"
+          path: "/etc/kubernetes"
```

Comment:
Why do we do this path change?

Reply:
@celebdor 
Hmm, I think that @bcrochet  reported in keepalived-unicast PR [1] that he needed to mount /etc/kubernetes instead of /etc/kubernetes/kubeconfig - so I did the same also here. 
I'll recheck this PR this time with /etc/kubernetes/kubeconfig, if it's works - I'll update the PR.
[1] 
https://github.com/openshift/machine-config-operator/pull/1504/files#diff-c2a4c0270459a7bea650ed87b237329eR22

Reply:
@celebdor - It appears that we don't need the kubeconfig anymore here, just the '/var/lib/kubelet' , so, I just deleted it and added the 'var/lib/kubelet'